crafters/image/ImageResizer/tests/test_imageresizer.py

```diff
@@ -52,6 +52,5 @@ def test_resize_gray(img_array):
     # Test for tuple/list target_size
     output_dim = (img_height, img_width)
     crafter = ImageResizer(target_size=output_dim)
-    img_array = create_random_gray_img_array(img_width, img_height)
     crafted_doc = crafter.craft(img_array)
```

Comment:
no need to craft again right?

Reply:
> no need to craft again right?

I am not sure... I would try to follow the old test structure.. I don' have any particula opinion

Reply:
it should be stateless so same input should lead to same output

Reply:
I don't undertand your proposal.. do you want to modify the asserts or do you want to remove one craft operation?

Reply:
one craft operation

Reply:
We can remove it in both tests... It should be the same structure for both test

Reply:
Sorry, I was confused, output_dim has changed